test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -644,6 +644,16 @@ def func(tensor):
         tensor = torch.view_as_complex(torch.randn(2, 1025, 400, 2))
         self._assert_consistency_complex(func, tensor)
 
+    def test_compute_compute_rtf_power(self):
+        def func(_):
+            channel = 4
+            psd_speech = torch.rand(129, channel, channel, dtype=torch.cfloat)
+            psd_noise = torch.rand(129, channel, channel, dtype=torch.cfloat)
+            return F.compute_rtf_power(psd_speech, psd_noise, 0)
+
+        dummy = torch.rand(1, 1)
+        self._assert_consistency(func, dummy)
```

Comment:
The use of dummy here seems it does not fulfill the purpose of the test.

Can you modify the `_assert_consistency` method so that it can accept multiple arguments, or pack two tensors with tuple?